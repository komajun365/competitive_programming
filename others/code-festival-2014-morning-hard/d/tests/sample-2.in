2 2 0 3 20 3
0 0
20 0
