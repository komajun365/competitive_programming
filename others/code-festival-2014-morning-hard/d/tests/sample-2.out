15.1961524227
